src/main.cpp

```diff
@@ -194,6 +194,10 @@ void InitRespendFilter() {
     doubleSpendFilter = CBloomFilter(MAX_DOUBLESPEND_BLOOM, 0.01, insecure_rand(), BLOOM_UPDATE_NONE);
 }
 
+bool IsStealthMode() {
+    return GetBoolArg("-stealth-mode", false);
```

Comment:
Suggest: -stealth or -stealthmode to be consistent with other comand-line args (which don't hyphenate)